### ISSUE TITLE
Error using MakeLoss or custom loss layer.

### ISSUE THREAD

**Issue by akaitsuki-ii** [dmlc/mxnet#4367]
I try calculate loss using a softmax_cross_entropy symbol with a MakeLoss symbol as the final output of my own network. After start training, the system returns following error info:

> Process finished with exit code 136 (interrupted by signal 8: SIGFPE)

However, when I replace the softmax_cross_entropy and MakeLoss with a custom loss layer (with input shape (batch_size, num_labels), return loss value in shape (1, )), same error occurs.
Is there any bugs with loss output?
